10.detect-separation/scripts/nbconverted/3.visualize-separation.r

```diff
@@ -0,0 +1,437 @@
+              width = 0.2) +
+  scale_fill_manual(labels = c("Amplified" = "Amplified",
+                               "Not Amplified" = "Not Amplified",
+                               "Unknown" = "Unknown"),
+                    values = c("Amplified" = "#d95f02",
+                               "Not Amplified" = "#1b9e77",
+                               "Unknown" = "grey25")) +
+  scale_color_manual(labels = c("Amplified" = "Amplified",
+                                "Not Amplified" = "Not Amplified",
+                                "Unknown" = "Unknown"),
+                     values = c("Amplified" = "#d95f02",
+                                "Not Amplified" = "#1b9e77",
+                                "Unknown" = "grey25")) +
+  theme_bw() +
+  ylab("VAE Feature 111 (k = 200)") +
+  xlab("") +
+  ggtitle("TARGET NBL MYCN Status") +
+  theme(axis.title.x = element_text(size = 7),
+       axis.title.y = element_text(size = 7),
+       axis.text.x = element_text(size = 6),
+       axis.text.y = element_text(size = 6),
+       plot.title = element_text(size = 8,
+                                 hjust = 0.5),
+       legend.position = "none")
+
+mycn_amp_gg
+
+file <- file.path("results", "mycn_nbl_scores.tsv")
+mycn_validation_df <- readr::read_tsv(file)
+
+mycn_validation_df$`MYCN status` <- dplyr::recode_factor(mycn_validation_df$`MYCN status`,
+                                                         "Non-amplified" = "Not Amplified")
+mycn_validation_df$`MYCN status` <- factor(mycn_validation_df$`MYCN status`,
+                                           levels = c("Amplified", "Not Amplified"))
+head(mycn_validation_df)
+
+mycn_validation_gg <-
+    ggplot(mycn_validation_df,
+           aes(y = vae_111,
+               x = `MYCN status`,
+               fill = `MYCN status`)) +
+  geom_boxplot(alpha = 0.5,
+               outlier.alpha = 0) +
+  geom_jitter(aes(color = `MYCN status`),
+              alpha = 0.7,
+              size = 0.9,
+              width = 0.2) +
+  theme_bw() +
+  scale_fill_manual(labels = c("Amplified" = "Amplified",
+                               "Not Amplified" = "Not Amplified"),
+                    values = c("Amplified" = "#d95f02",
+                               "Not Amplified" = "#1b9e77")) +
+  scale_color_manual(labels = c("Amplified" = "Amplified",
+                                "Not Amplified" = "Not Amplified"),
+                     values = c("Amplified" = "#d95f02",
+                                "Not Amplified" = "#1b9e77")) +
+  ylab("VAE Feature 111 (k = 200)") +
+  xlab("") +
+  ggtitle("NBL Cell Lines MYCN Status") +
+  theme(axis.title.x = element_text(size = 7),
+       axis.title.y = element_text(size = 7),
+       axis.text.x = element_text(size = 6),
+       axis.text.y = element_text(size = 6),
+       plot.title = element_text(size = 8,
+                                 hjust = 0.5),
+       legend.position = "none")
+
+mycn_validation_gg
+
+# Plot multi panel figure
```

Comment:
I see there is a TARGET NBL box plot and NBL box plot.  Is the the NBL plot from normal patients?

Reply:
The TARGET boxplot is from patients, the other is from Cell lines

Reply:
The cell line scores were derived from the signature learned from TARGET data